.github/workflows/js-test-and-release.yml

```diff
@@ -27,74 +27,35 @@ jobs:
     strategy:
       matrix:
         os: [windows-latest, ubuntu-latest, macos-latest]
-        node: [16]
-      fail-fast: true
+      fail-fast: false
     steps:
       - uses: actions/checkout@v3
       - uses: actions/setup-node@v3
         with:
-          node-version: ${{ matrix.node }}
+          node-version: lts/*
```

Comment:
I believe it will be overwritten: 
* when repo is added to unified CI (this repo has already been)
https://github.com/protocol/.github/blob/30836503d140fec6454ef6f49d96372a9167757f/.github/actions/add-unified-ci-to-new-go-repos/src/main.ts
* copy workflow from .github repo: https://github.com/protocol/.github/blob/b1a2633be4742ee57b26f27ef475d5227b585476/.github/workflows/copy-workflow.yml
* code where unified CI compares existing workflows to templated workflows in .github repo: https://github.com/protocol/.github/blob/b1a2633be4742ee57b26f27ef475d5227b585476/.github/workflows/copy-workflow.yml#L115-L124

Reply:
@galargh can you confirm?

Reply:
Yes, it's going to get overwritten. Whenever https://github.com/protocol/.github/actions/workflows/dispatch.yml runs, it updates uCI files in all the repositories where it's configured (if needed of course). That's why files managed by uCI have this header: https://github.com/ipfs/js-kubo-rpc-client/blob/b3da286f676c905eb3b4978d0e20ee449926df8e/.github/workflows/js-test-and-release.yml#L1-L2

Here's an automated PR that wants to update the workflow here: https://github.com/ipfs/js-kubo-rpc-client/pull/85 (it would have been automerged if only `test-node` on windows passed).

If there's a change we want to introduce that would benefit all JS repos participating in uCI, then this would be the place to do it: https://github.com/protocol/.github/blob/master/templates/.github/workflows/js-test-and-release.yml

_Side note_: the way [Unified CI](https://github.com/protocol/.github) provisions files is different from how [GitHub Management](https://github.com/ipfs/github-mgmt) does it.  For example, if we were to modify https://github.com/ipfs/js-kubo-rpc-client/blob/master/.github/workflows/stale.yml which is managed here https://github.com/ipfs/github-mgmt/blob/eed32757b57354659c07d738d36ecfbc25a033ef/github/ipfs.yml#L4589-L4591, then the next [sync](https://github.com/ipfs/github-mgmt/actions/workflows/sync.yml) would have noticed that the two diverged and would update the `content` in the YAML config.

Reply:
So if a project needs to disable node LTS or other node versions for some reason, what is the best way to accomplish that?

Can we make the unified CI workflow dynamic with a default to LTS? I'm not sure the best way to accomplish this but it would allow individual projects to update/override when necessary. Without being overwritten.